cli/script/command-executor.ts

```diff
@@ -493,13 +503,13 @@ function getTotalActiveFromDeploymentMetrics(metrics: DeploymentMetrics): number
     return totalActive;
 }
 
-function initiateExternalAuthenticationAsync(serverUrl: string, action: string): void {
+function initiateExternalAuthenticationAsync(action: string, serverUrl?: string): void {
     var message: string = `A browser is being launched to authenticate your account. Follow the instructions ` +
         `it displays to complete your ${action === "register" ? "registration" : "login"}.\r\n`;
 
     log(message);
     var hostname: string = os.hostname();
-    var url: string = serverUrl + "/auth/" + action + "?hostname=" + hostname;
+    var url: string = (serverUrl || AccountManager.SERVER_URL) + "/auth/" + action + "?hostname=" + hostname;
```

Comment:
I think the rule should be that you fix whatever you touch, so I've gone ahead and converted the relevant strings, including this one!